src/content/guides/using-persistent-volumes-with-csi-driver-on-aws/index.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Using persistent volumes with the EBS CSI driver on AWS
+description: Tutorial on how to use dynamically provisioned persistent volumes with the EBS CSI driver on a cluster running on Amazon Web Services (AWS)."
+type: page
+weight: 50
+tags: ["tutorial"]
+user_questions:
+  - How do I install the EBS CSI driver?
+  - How do I provision an EBS instance on AWS?
+  - How do I test the EBS CSI storage class?
+  - How do I use EBS CSI volumes?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-firecracker
+---
+
+# Using persistent volumes with the EBS CSI driver on AWS
+
+The Container Storage Interface (CSI) has been promoted to general availability (GA) in Kubernetes v1.13 and is becoming the standard to replace the current Kubernetes `in-tree` storage plugin to handle volumes for different providers.
+
+The aws-ebs-csi-driver-app, offered via our [App Platform](/basics/app-platform/) in the Giant Swarm Playground catalog, provides a CSI interface to manage the lifecycle of Amazon EBS volumes.
+
+## Installing the EBS CSI driver
+
+To install the EBS CSI driver you will need to follow these steps:
+
+1. Access the [web interface](/reference/web-interface/) and select the cluster on which you want to install the EBS CSI driver.
+2. Open the Apps tab.
+3. Click the Install App button
+4. Select the Giant Swarm Playground catalog.
+5. Select the App named aws-ebs-csi-driver-app.
+6. Click the Configure & Install button. Make sure that the correct cluster is selected.
+7. Click the Install App button.
+
+## Storage classes
+
+Once you installed the `aws-ebs-csi-driver-app` from _Giant Swarm Playground_ catalog, your Kubernetes cluster will have a Storage Class `ebs-csi` deployed.
+
+The storage class offers volume encryption and allows resizing by default.
+
+## Creating persistent volumes
+
+The most straight forward way to create a persistent volume is to create a [PersistentVolumeClaim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) resource, which will automatically create a corresponding `PersistentVolume` (PV) resource for you.
```

Comment:
might be worth mentioning also here that you need to configure storage class on that PVC< it can be seen in the example below but its not explicitly said here which might be confusing